invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/intro_section.html

```diff
@@ -0,0 +1,93 @@
+{#
+  Copyright (C) 2019-2022 CERN.
+
+  Invenio App RDM is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+<section class="light-bg rel-pt-3 rel-pb-3">
+  {{ records }}
+  <h2 class="section-title">
+    {{ _("You've successfully installed InvenioRDM!") }}
+    <br/>
+    {{ _("What is ahead?") }}
+  </h2>
+
+  <div class="ui link cards centered container">
+    <a href="https://inveniordm.docs.cern.ch/install/configuration/" target="_blank" class="ui card">
+      <div class="image">
+        <div class="featured-list">1</div>
+      </div>
+      <div class="content">
+        <div class="header">
+          {{ _("Configure") }}
+        </div>
+        <div class="description">
+          {% trans %}
+            Configure the datamodel, resource types, permissions and much more to make it suit your needs!
```

Comment:
Are these changes in text needed? We are changing the translations too and it will interfere with the current translation process that is undergoing for V10.1...